### PR TITLE
Changed puchdb Version to 7.0.0 

### PR DESCRIPTION
**Pull request by frankbauer**
Fixed install and build errors with node v10.11.0 on MacOS Mojave.

### CODE REVIEW

File: webapp/src/db.ts [hunk 1]
```diff
@@ -4,7 +4,9 @@ import * as Promise from "bluebird";
 
 const PouchDB = require("pouchdb");
 /* tslint:disable:no-submodule-imports TODO(tslint) */
-require('pouchdb/extras/memory');
+PouchDB.plugin(require('pouchdb-adapter-cordova-sqlite'));
```
Comment (pelikhan) [inline webapp/src/db.ts]: Do we need these?

Comment (frankbauer) [reply webapp/src/db.ts]: So far it builds and runs fine without those modules. I included it since it was part of the original pouchdb.

**Comment by pelikhan** [comment]
I think we can (want) drop those if we can :)

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by frankbauer** [comment]
I think we can remove the dependencies as in `package.json` as well. I can build another clean pull request if that is ok?

**Comment by pelikhan** [comment]
You could just update this one.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pelikhan** [comment]
Looks like PouchDB is not supported on IE anymore. Needs to revert this.

**Comment by frankbauer** [comment]
Hm, maybe we need the websql plugin for this case:

> PouchDB supports all modern browsers, using IndexedDB under the hood and falling back to WebSQL where IndexedDB isn't supported.
(from [pouchdb.com](https://pouchdb.com/learn.html#browser_support))

Unfortunately I can not test this.

**Pull request merged**